autoload/wintabs/ui.vim

```diff
@@ -176,14 +203,18 @@ function! s:get_spaceline()
   let line = g:wintabs_ui_sep_spaceline
   let length = 1
   for tab in range(1, spaces)
-    " get and normalize space name
-    let name = ' '.tab.' '
-    let length += len(name)
+    " get tab name
+    let name = s:get_tab_name(tab)
 
     " highlight current space
     if tab == tabpagenr()
+      let name = g:wintabs_ui_active_lsepgroup.name.g:wintabs_ui_active_rsepgroup
       let name = '%#'.g:wintabs_ui_active_higroup.'#'.name.'%##'
     endif
+    let name = ' '.name.' '
+
+    " get and normalize space name
+    let length += len(name)
```

Comment:
this must happen between adding separator and adding highlight group name, as the highlight group shouldn't be counted towards character length